democracy/src/benchmarking.rs

```diff
@@ -0,0 +1,91 @@
+use crate::{Pallet as EncointerDemocracy, *};
+use codec::Encode;
+use encointer_primitives::{
+	ceremonies::Reputation,
+	communities::CommunityIdentifier,
+	democracy::{ProposalState, Tally, Vote},
+	storage::{current_ceremony_index_key, global_reputation_count, participant_reputation},
+};
+use frame_benchmarking::{account, benchmarks, impl_benchmark_test_suite};
+use frame_support::{assert_ok, traits::OnInitialize, BoundedVec};
+use frame_system::RawOrigin;
+#[cfg(not(feature = "std"))]
+use sp_std::vec;
+
+fn advance_n_blocks<T: Config>(n: u64) {
+	for _ in 0..n {
+		frame_system::Pallet::<T>::set_block_number(
+			frame_system::Pallet::<T>::block_number() + 1u32.into(),
+		);
+		frame_system::Pallet::<T>::on_initialize(frame_system::Pallet::<T>::block_number());
+	}
+}
+
+benchmarks! {
+
+	submit_proposal {
```

Comment:
Is the number of active proposals bounded?
It may make sense to restrict the number of proposals per reputable to one per cycle. but that can happen in another PR

Reply:
no it is not bounded. i added this to https://github.com/encointer/pallets/issues/347